app/models/block/document_list.rb

```diff
@@ -0,0 +1,26 @@
+module Block
+  class DocumentList < Block::Base
+    attr_reader :items
+
+    def initialize(block_hash)
+      super(block_hash)
+
+      @items = data.fetch("items").map do |i|
```

Comment:
Minor, and not a blocker, but as it's doing so much, I'd have made items into a method.

Reply:
Good shout, have converted 👍